Java/src/main/java/com/gildedrose/item/Sulfuras.java

```diff
@@ -0,0 +1,12 @@
+package com.gildedrose.item;
+
+public class Sulfuras implements CustomisedItem {
+
+    public Item item;
+
+    public Sulfuras(Item item) {
```

Comment:
If you don't have any logic in the constructor, you can just remove it and Java will use the default constructor.